cypress/e2e/GUI/empty_box.cy.js

```diff
@@ -0,0 +1,13 @@
+describe('Exibição de mensagem de clientes ausentes', () => {
+    beforeEach(() => {
+      cy.intercept('GET', 'http://localhost:3001/customers?page=1&limit=10&size=All', { statusCode: 200, body: `{customers:[]}` }).as('getCustomers');
+    });
+
+    it('Deve exibir a mensagem de "No customers available." quando não há clientes', () => {
+      cy.visit('/');
+
+      cy.get('span').should('exist');
```

Comment:
Que tal verificar a imagem do tipo SVG?

E verificar a visibilidade da mesma?

```suggestion
      cy.get('svg').should('be.visible');
```

Às vezes, elementos podem existir no DOM, mas não estarem visíveis.
Portanto, dê preferência por verificar pela visibilidade dos mesmos.